front/src/app/services/project.service.ts

```diff
@@ -1,9 +1,37 @@
 import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Project } from '../models/project';
+import { environment } from '../../environments/environment';
+import { map } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ProjectService {
+  projectUrl = environment.url + '/projects';
+  addUrl = environment.url + '/projects_add_all';
 
-  constructor() { }
+  constructor(private http: HttpClient) { }
+
+  fetchProjects(email: string) {
+    const httpOptions = {
+      headers: new HttpHeaders({
+        'Content-Type':  'application/json',
+        Email: email
+      })
+    };
+    return this.http.get<Project[]>(this.projectUrl, httpOptions).pipe(map(projects => projects as Project[]));
+  }
```

Comment:
Domyślam się, że tutaj ten nagłówek jest załączony na potrzeby dema zamiast tokena, pamiętajcie żeby to później zniknęło.